docs/development.md

```diff
@@ -17,6 +17,12 @@ _WARNING:_ MoonZoon is in the phase of early development and a CI pipeline / lin
   cargo install cargo-make --no-default-features
   makers -V # cargo-make 0.37.12
   ```
+  
+- [Rust Nightly Toolchain]
+  ```bash
+  rustup install nightly-x86_64-unknown-linux-gnu
+  rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
+  ```
```

Comment:
Please remove it, I assume it has been added here by accident.

Reply:
It has been removed